core/src/main/java/org/apache/accumulo/core/classloader/URLContextClassLoaderFactory.java

```diff
@@ -56,6 +56,12 @@ public URLContextClassLoaderFactory() {
     }
   }
 
+  @Override
+  @SuppressWarnings("deprecation")
+  public final void finalize() {
+    // Prevent finalizer attacks (SpotBugs CT_CONSTRUCTOR_THROW)
+  }
+
```

Comment:
Is this necessary if the class is final? It seems one OR the other is necessary to fix this. We probably don't need to add the finalize method if it is fixed another way.